src/main/resources/swagger.api/mod-search.yaml

```diff
@@ -0,0 +1,143 @@
+openapi: 3.0.0
+info:
+  title: Search API
+  version: v1
+
+servers:
+  - url: /search/
+
+paths:
+  /instances:
+    get:
+      operationId: searchInstances
+      description: Get a list of instances for CQL query
+      parameters:
+        - $ref: '#/components/parameters/search-cql-query'
+        - $ref: '#/components/parameters/search-pageable-limit'
+        - $ref: '#/components/parameters/search-pageable-offset'
+        - $ref: '#/components/parameters/x-okapi-tenant-header'
+
+      responses:
+        '200':
+          description: 'Search response with founded instances'
```

Comment:
Maybe change to `Search response with instances found` or `Instance search result`?